.github/workflows/ci.yml

```
@@ -154,6 +154,9 @@ jobs:
             - { target: x86_64-linux-android,             os: ubuntu-latest,  cpp: 1,           std: 1, run: 1 }
             - { target: x86_64-pc-windows-gnu,            os: ubuntu-latest,  cpp: 1,           std: 1, run: 1 }
             - { target: i686-pc-windows-gnu,              os: ubuntu-latest,  cpp: 1,           std: 1, run: 1 }
+            # Disabled, see rust-lang/rust#85821
```

this could be a full link

```suggestion
            # Disabled, see https://github.com/rust-lang/rust/issues/85821
```